config/installation_config.yml

```diff
@@ -41,67 +56,138 @@
   display_title: 'Chatwoot Metadata'
   description: 'Display default Chatwoot metadata like favicons and upgrade warnings'
   type: boolean
-- name: MAILER_INBOUND_EMAIL_DOMAIN
-  value:
-  locked: false
-- name: MAILER_SUPPORT_EMAIL
-  value:
+# ------- End of Branding Related Config ------- #
+
+
+
+# ------- Signup & Account Related Config ------- #
+- name: ENABLE_ACCOUNT_SIGNUP
+  display_title: 'Enable Account Signup'
+  type: boolean
+  description: 'Allow users to signup for new accounts'
+  value: false
+  description: 'Allow users to signup for new accounts'
   locked: false
 - name: CREATE_NEW_ACCOUNT_FROM_DASHBOARD
   value: false
+  description: 'Allow users to create new accounts from the dashboard'
   locked: false
-- name: INSTALLATION_EVENTS_WEBHOOK_URL
+- name: HCAPTCHA_SITE_KEY
```

Comment:
Does changing the order affect us any way?

Reply:
Not in the case of installation config. we take this as array of objects and process accordingly.